components/binding-usage-controller/Jenkinsfile

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env groovy
-
 def label = "kyma-${UUID.randomUUID().toString()}"
 def application = "binding-usage-controller"
 def dockerPushRoot = "${env.DOCKER_REGISTRY}${params.PUSH_DIR}"
 def dockerImageTag = params.APP_VERSION
+def registry = env.DOCKER_REGISTRY
+def buildpack = 'golang-buildpack:0.0.9'
+def appFolder = env.APP_FOLDER
+def repositoryName = 'kyma'
+
+def workDir = ""
+def checkoutDir = ""
```

Comment:
why it cannot by simply inlined here?
you are creating it by concatenating hardcoded value + env variable defined above.
```
checkoutDir = "src/github.com/kyma-project/$repositoryName"
```

I do not see any reason to have it defined in the setup stage

same story for the componentDir variable

Reply:
Done